plugins/serialize-disabled-interactions/README.md

```diff
@@ -43,39 +73,19 @@ All of the keys are optional. For example, if all of the blocks on the workspace
 are deletable, the `'notDeletable'` key will be undefined. If all of the keys
 are undefined, then the `'disabledInteractions`' key is also undefined.
 
-## Migration Guide
-
-This plugin was created because the old XML serialization system supported
-serializing these attributes, but the new JSON system does not. For more
-information about why this is, see [this forum post](https://groups.google.com/g/blockly/c/eP9PXVfCaHs/m/S4rPmwnTAQAJ).
-
-Usually these values don't need to be serialized. Instead you can just set these
-attributes in your block definition:
-```javascript
-Blockly.Blocks['my_start_block'] = {
-  init: function() {
-    // Your normal init code...
-    this.setDeletable(false);
-    this.setMovable(false);
-    this.setEditable(false);
-  }
-}
-```
+## Usage for Blockly < v9.3.0
```

Comment:
I think you should just delete this section (and the installation section) for two reasons:
- For Blockly < 9.3 they'll need to use an old version of this plugin. If they insist on doing that, they can look at the old version of the code for instructions too.
- I want to strongly discourage people from using it, because it is being deprecated. I think giving people easy instructions on how to install and use a deprecated thing sends mixed messages.